tests/test_ophys/test_tools_roiextractors.py

```diff
@@ -1054,6 +1055,53 @@ def test_add_fluorescence_traces_regular_timestamps(self):
             self.assertEqual(roi_response_series[series_name].starting_time, times[0])
             self.assertEqual(roi_response_series[series_name].timestamps, None)
 
+    def test_add_fluorescence_traces_irregular_default_timestamps(self):
+        """Test adding traces with default timestamps."""
+
+        segmentation_extractor = generate_dummy_segmentation_extractor(
+            num_rois=2,
+            num_frames=5,
+            num_rows=self.num_rows,
+            num_columns=self.num_columns,
+        )
+
+        add_fluorescence_traces(
+            segmentation_extractor=segmentation_extractor,
+            nwbfile=self.nwbfile,
+            metadata=self.metadata,
+        )
+
+        ophys = get_module(self.nwbfile, "ophys")
+        roi_response_series = ophys.get(self.fluorescence_name).roi_response_series
+        for series_name in roi_response_series.keys():
+            self.assertEqual(roi_response_series[series_name].rate, None)
```

Comment:
I don't understand this test. The dummy segmentation extractor has a constant sampling rate (30 by default). What do you mean by default timestamps in the string?

Reply:
I think some code might have been lost in between pushes, updated it to what I think is the expected behavior (but again, I don't thing any `rate` info should be in overhead metadata, should just be attached to the extractor object